### DISCUSSION TITLE
Better summary ad-hoc statistics for parsed numeric log fields

### DISCUSSION THREAD

**Discussion by hmoravec** [grafana/grafana#56846] [Observability]
**What would you like to be added**:
Ad-hoc statistics for numeric log fields show counts of the values for each value. There should be rather histogram with summary statistics (e.g. min, max, average, median, deviation).
![Screenshot from 2020-02-03 00-16-33](https://user-images.githubusercontent.com/43935406/73617087-6cb3b780-461b-11ea-9c5d-b6bfe9563a34.png)

**Why is this needed**:
Current ad-hoc statistics for numeric log fields are not useful.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
